### ISSUE TITLE
fit_text raises IndexError when text_frame has empty string

### ISSUE THREAD

**Issue by yirey** [scanny/python-pptx#260]
```python
tf = slide.shapes.placeholders[0].text_frame
tf.text = ''
tf.fit_text(max_size=40)
```

'IndexError: pop from empty list' arises if there is no text

```
../../../anaconda3/lib/site-packages/pptx/text/text.py:79: in fit_text
    font_family, max_size, bold, italic, font_file
../../../anaconda3/lib/site-packages/pptx/text/text.py:223: in _best_fit_font_size
    self.text, self._extents, max_size, font_file
../../../anaconda3/lib/site-packages/pptx/text/layout.py:29: in best_fit_font_size
    return text_fitter._best_fit_font_size(max_size)
../../../anaconda3/lib/site-packages/pptx/text/layout.py:40: in _best_fit_font_size
    return sizes.find_max(predicate)
../../../anaconda3/lib/site-packages/pptx/text/layout.py:131: in find_max
    if predicate(self.value):
../../../anaconda3/lib/site-packages/pptx/text/layout.py:81: in predicate
    text_lines = self._wrap_lines(self._line_source, point_size)
../../../anaconda3/lib/site-packages/pptx/text/layout.py:109: in _wrap_lines
    text, remainder = self._break_line(line_source, point_size)
../../../anaconda3/lib/site-packages/pptx/text/layout.py:48: in _break_line
    lines = _BinarySearchTree.from_ordered_sequence(line_source)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <class 'pptx.text.layout._BinarySearchTree'>, iseq = <_LineSource('')>

    @classmethod
    def from_ordered_sequence(cls, iseq):
        """
            Return the root of a balanced binary search tree populated with the
            values in iterable *iseq*.
            """
        seq = list(iseq)
        # optimize for usually all fits by making longest first
>       bst = cls(seq.pop())
E       IndexError: pop from empty list

../../../anaconda3/lib/site-packages/pptx/text/layout.py:148: IndexError
```

**Comment by scanny** [comment]
Thanks @yirey, this is a useful report, I'll add a test case to cover this situation.

**Comment by scanny** [comment]
Fixed in release v0.6.12 circa Aug 11 2018.
